core/src/main/java/com/backblaze/b2/client/structures/B2StartLargeFileRequest.java

```diff
@@ -138,6 +138,10 @@ public Builder setCustomField(String name, String value) {
             return this;
         }
 
+        public Builder setLargeFileSha1(String largeFileSha1) {
```

Comment:
This is somewhat unrelated, but we talked about it being nice to have included.

Reply:
yep.  thx.